boards/px4/fmu-v5/stackcheck.cmake

```diff
@@ -20,7 +20,7 @@ px4_add_board(
 	DRIVERS
 		adc
 		barometer # all available barometer drivers
-		batt_smbus
+		#batt_smbus
```

Comment:
How does this relate to the rest of the PR?

Reply:
This change pushed the px4_fmu-v5_stackcheck build over the flash limit. NuttX stackcheck (https://nuttx.org/doku.php?id=wiki:howtos:run-time-stack-checking) instruments every single function call, so in this particular case my guess is it breaks the inlining in these param wrappers. https://github.com/PX4/Firmware/blob/34b03d56593815e16a29ba5dcef9b66208285c98/src/platforms/px4_param.h#L123